### PR TITLE
Add metrics

### PR DESCRIPTION
**Pull request by erikh360**
also fixed some warnings in the tests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,8 +9,18 @@ RUN mix local.rebar --force
 RUN mix deps.get
 RUN mix deps.compile
 
+FROM elixir:1.9-alpine
+ENV MIX_ENV="prod"
+RUN mix local.hex --force
+RUN mix local.rebar --force
```
Comment (rudigiesler) [inline Dockerfile]: Do we need to run these two mix commands, if we're copying everything from `elixir`? Same with `mix compile`, could that be moved to `elixir`?

**Review by rudigiesler — APPROVED** [review]
Just one small thing, but it looks good

File: Dockerfile (hunk 1 above)
Comment (erikh360) [reply Dockerfile]: Let me try that

Comment (erikh360) [reply Dockerfile]: `mix compile` is different from `mix deps.compile` so I'll that one

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**